Cargo.toml

```diff
@@ -16,3 +23,59 @@ members = [
 	"ethereum-types",
 	"ethbloom",
 ]
+
+[workspace.dependencies]
+serde_json = "1.0.41"
+criterion = "0.5.1"
+rand = { version = "0.8.0", default-features = false }
+hex-literal = "0.4.1"
+scale-info = { version = ">=1.0, <3", default-features = false }
+quickcheck = "1"
+rustc-hex = { version = "2.0.1", default-features = false }
+static_assertions = "1.0.0"
+arbitrary = "1.0"
+tiny-keccak = "2.0"
+crunchy = { version = "0.2.2", default-features = false }
+serde = { version = "1.0.101", default-features = false }
+codec = { package = "parity-scale-codec", version = "3.7.4", default-features = false }
+log = { version = "0.4.17", default-features = false }
+schemars = ">=0.8.12"
+rand_xorshift = "0.3.0"
+tempfile = "3.1.0"
+smallvec = "1.0.0"
+parking_lot = "0.12.0"
+num_cpus = "1.10.1"
+regex = "1.3.1"
+rocksdb = { version = "0.23.0", default-features = false }
```

Comment:
I'd prefer any non-refactoring changes to be in a separate PR (please also adjust the relevant changelog, e.g. is this a breaking change?).

Reply:
Do you accept the Cargo.toml refactor? if so, I'm leaving the refactor, and making a new PR for rocksdb upgradation

Reply:
BTW: This is not a breaking change, the Cargo.toml refactor should only change the style, and I don't see https://github.com/rust-rocksdb/rust-rocksdb/releases/tag/v0.23.0 has breaking change

Reply:
@jasl It's hard to review which dependencies you have bumped or not. It would be great if you could split this PR in two different ones:

1. Move to dependencies to cargo workspace i.e, refactoring
2. Bump rockdb dependency

Reply:
I rollback the rocksdb version bump